### PR TITLE
inline onclick updated with onmousedown #78 

### PR DESCRIPTION
**Pull request by diliger**
following KitchenSink examples
#78

### CODE REVIEW

File: src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
 <template>
     <template is="dom-bind">
         <template is="dom-repeat" items="{{model.Illustrations}}" restamp="true">
-            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onclick="++this.value;" value="{{item.Select$::click}}" />
+            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onmousedown="++this.value;" value="{{item.Select$::click}}" />
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html]: The above line still does not work in Firefox

File: src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <template>
     <template is="dom-bind">
         <template is="dom-repeat" items="{{model.Illustrations}}" restamp="true">
-            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onclick="++this.value;" value="{{item.Select$::click}}" />
+            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onmousedown="++this.value;" value="{{item.Select$::click}}" />
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html]: The above line still does not work in Firefox

File: src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html [hunk 3]
```diff
@@ -3,16 +3,16 @@
 <template>
     <template is="dom-bind">
         <template is="dom-repeat" items="{{model.Illustrations}}" restamp="true">
-            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onclick="++this.value;" value="{{item.Select$::click}}" />
+            <img slot="images/thumbnail" src="{{item.PreviewURL}}" on-click="onPreviewClick" value="{{item.Select$::click}}" />
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html]: I like this change, but now `value="{{item.Select$::click}}"` is not needed here. Please remove that.

File: src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html [hunk 4]
```diff
@@ -1,12 +1,22 @@
 <template>
     <template is="dom-bind">
         <template is="dom-repeat" items="{{model.Illustrations}}" restamp="true">
-            <img slot="images/thumbnail" src="{{item.PreviewURL}}" onclick="++this.value;" value="{{item.Select$::click}}" />
+            <img slot="images/thumbnail" src="{{item.PreviewURL}}" on-click="onPreviewClick" value="{{item.Select$::click}}" />
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html]: I like this change, but now `value="{{item.Select$::click}}"` is not needed here. Please remove that.

File: src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
             <starcounter-include partial="{{model.Selected.Content}}"></starcounter-include>
         </div>
     </template>
+    <script>
+        (function () {
+            var script = document._currentScript || document.currentScript;
+            var template = script.previousElementSibling;
+            
+            template.onPreviewClick = function (e) {
+                template.set("model.item.Select$", ++e.model.item.Select$);
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/IllustrationsPage.html]: Better do `template.set("model.item.Select$", ++template.get("model.item.Select$"));`

File: src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html [hunk 6]
```diff
@@ -22,6 +22,10 @@
             var script = document._currentScript || document.currentScript;
             var template = script.previousElementSibling;
 
+            template.onPreviewClick = function (e) {
+                template.set("model.item.Select$", ++e.model.item.Select$);
```
Comment (warpech) [inline src/Images/wwwroot/Images/viewmodels/EditableIllustrationsPage.html]: Better do `template.set("model.item.Select$", ++template.get("model.item.Select$"));`

File: src/Images/ViewModels/EditableContentPage.json.cs [hunk 7]
```diff
@@ -8,12 +8,12 @@ partial class EditableContentPage : Json, IBound<Content>
     {
         protected IllustrationHelper Helper = new IllustrationHelper();
         protected List<string> OldUrls = new List<string>();
-        public string SessionId => Session.Current?.SessionId;
+        //public string SessionId => Session.Current?.SessionId;
```
Comment (warpech) [inline src/Images/ViewModels/EditableContentPage.json.cs]: Should this really be commented out? `SessionId` is used in `EditableContentPage.html`

**Comment by diliger** [comment]
Fixed last comments.

**Pull request merged**